main_static.go

```diff
@@ -0,0 +1,17 @@
+//go:build linux
+// +build linux
+
+package main
+
+import (
+	_ "time/tzdata"
+
+	_ "github.com/breml/rootcerts"
```

Comment:
I get your idea to embed all external dependencies like tzdata and root certs and I like it. But I have some doubts about security and certificate handling. Can user import own certificates? What is about certificate revocation? Do we need to release in shorter periods?

Reply:
Currently go will lock on the container file system too and include existing certificates if they are in the correct folder.
This can be changed but i thought it is usefull if own certificates need to be added.
There are som infos https://github.com/breml/rootcerts in the readme.
As the revocation of rootcertificates are somwhat rare the release cyclus can stay the same.

Reply:
> Currently go will lock on the container file system too and include existing certificates if they are in the correct folder. 

Ok, that means, if someone uses the binary, it is still possible to import own certificates. In docker it will be also possible if user mounts certificate direction to the right place?

Reply:
> > Currently go will lock on the container file system too and include existing certificates if they are in the correct folder.
> 
> Ok, that means, if someone uses the binary, it is still possible to import own certificates. In docker it will be also possible if user mounts certificate direction to the right place?

If I got the description correct this should be the case.
This also applies only to the root certificates, are there usecases where custom root(not intermediate) certificates are used?

Reply:
removed static certs in favor of cert file(same as alpine)